themes/src/main/resources/theme/keycloak/login/resources/css/login.css

```diff
@@ -574,6 +574,81 @@ ul#kc-totp-supported-apps {
     width: 40%;
 }
 
+/* Internet Explorer 11 compatibility workaround for select-authenticator screen */
+@media all and (-ms-high-contrast: none),
+(-ms-high-contrast: active) {
+    .select-auth-box-parent:hover .select-auth-box-headline,
+    .select-auth-box-parent:hover .select-auth-box-icon,
+    .select-auth-box-parent:hover .select-auth-box-arrow,
+    .select-auth-box-parent:hover .select-auth-box-desc {
+        color: #06c;
+    }
+    .select-auth-box-parent {
+        border-top: 1px solid #f0f0f0;
+        padding-top: 1rem;
+        padding-bottom: 1rem;
+        cursor: pointer;
+    }
+    .select-auth-box-parent:hover {
+        border-top: 1px solid #06c;
+    }
+
+    .select-auth-box-headline {
+        font-size: 16px;
+        font-weight: bold;
+    }
+
+    .select-auth-box-desc {
+        font-size: 14px;
+    }
+}
+/* End of IE11 workaround for select-authenticator screen */
+
+.select-auth-box-arrow{
+    display: flex;
+    align-items: center;
+    margin-right: 2rem;
+}
+
+.select-auth-box-icon{
+    display: flex;
+    flex: 0 0 2em;
+    justify-content: center;
+    margin-right: 1rem;
+    margin-left: 3rem;
+}
+
+.select-auth-box-parent:hover .select-auth-box-headline,
+.select-auth-box-parent:hover .select-auth-box-icon,
+.select-auth-box-parent:hover .select-auth-box-arrow,
+.select-auth-box-parent:hover .select-auth-box-desc{
+    color: var(--pf-global--primary-color--100);
+}
+
+.select-auth-box-parent{
+    border-top: 1px solid var(--pf-global--palette--black-200);
+    padding-top: 1rem;
+    padding-bottom: 1rem;
+    cursor: pointer;
+}
+
+.select-auth-box-parent:hover{
+    border-top: 1px solid var(--pf-global--primary-color--100);
```

Comment:
This is very subjective (as usually with UI stuff ;)) and it's definitely not a blocker but I think it looks a bit awkward that the blue border is just on top. Maybe it'd look better if it would be at the bottom as well? I don't know... Is there any PF4 best practice on this?

Reply:
@vmuzikar: Martin asked me the exact same question couple days ago. I remember that during the initial polishing in May I removed the horizontal bottom line based on the comment[1] from @stianst but I left the top horizontal line based on PF guidelines[2]:
![image](https://user-images.githubusercontent.com/12138171/98156007-771b1e80-1ed7-11eb-9dd0-38fb0ccdd3b5.png)

So I don't know now, any thoughts?

[1] https://issues.redhat.com/browse/KEYCLOAK-12824?focusedCommentId=14084158&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-14084158
[2] https://www.patternfly.org/v4/components/login-page/design-guidelines/

Reply:
I wouldn't add the bottom line to normal (non-hover) state to keep it compliant with the [PF4 guidelines](https://www.patternfly.org/v4/images/multi-factor.1cc14528bafc7a0cac690fb3105ee2e1.png).

So maybe we could take some other approach for the hover effect (which is not covered in the guideline if I'm correct). What about adding a light blue background color to the button on hover (instead of changing the line and text color)? WDYT @mabartos @Pepo48?

Reply:
@vmuzikar (Cc: @mabartos) what about this approach?
![image](https://user-images.githubusercontent.com/12138171/100010819-23626d80-2dd1-11eb-9cb6-c8cb3a1c344d.png)
I pushed it here in a separate commit to make the changes more visible, I will squash it once we agree on the final design :)
https://github.com/keycloak/keycloak/pull/7488/commits/ccca13f8d6331a59829fdf038ea29f8a196ea98a#diff-cc2b625f843865ee5186a5b77258a0b2fb66c0754b9e7e2be0bd0247408015e2

Reply:
@vmuzikar @mabartos Haley from PF team suggested that we might to take inspiration from the hover effect of selectable-tabs PF component [1]. But I guess it's up to us to make the final decision :)

[1] https://www.patternfly.org/v4/components/data-list/#selectable-rows

Reply:
@Pepo48 I'm afraid it's not entirely up to us. :( We need to stick whatever PF suggests.